ci/scripts/driver.sh

```diff
@@ -116,27 +98,36 @@ for pr in ${pr_list}; do
   mkdir -p "${pr_dir}"
   # call clone-build_ci to clone and build PR
   id=$("${GH}" pr view "${pr}" --repo "${repo_url}" --json id --jq '.id')
-  "${script_pwd}/clone-build_ci.sh" -p "${pr}" -d "${pr_dir}" -o "${pr_dir}/output_${id}"
+  "${WF_ROOT_DIR}/ci/scripts/clone-build_ci.sh" -p "${pr}" -d "${pr_dir}" -o "${pr_dir}/output_${id}"
   ci_status=$?
   if [[ ${ci_status} -eq 0 ]]; then
     #setup runtime env for correct python install
-    module use "${pr_dir}"/global-workflow/modulefiles
-    module load "module_base.${TARGET}"
-    cd "${pr_dir}/global-workflow/ush/python/pygw"
-    python3 -m pip install .
-    export PYTHONPATH="${pr_dir}/global-workflow/workflow" # Needed becuase Host class is in workflow 
-    module load rocoto
+    export HOMEGFS="${pr_dir}/global-workflow"
+    module use "${HOMEGFS}/modulefiles"
+    module load "module_setup.${TARGET}"
+    module list
+    #setup space to put an experiment
     export RUNTEST="${pr_dir}/RUNTEST"
+    rm -Rf "${RUNTEST}"
     mkdir -p "${RUNTEST}"
-    cd "${RUNTEST}"
-    rm -Rf ./*
-    export HOMEGFS="${pr_dir}/global-workflow"
-    "${script_pwd}/create_experment.py" --yaml "${script_pwd}/cold_96_00z.yaml"
+    # TODO adding pip install for YAML until is in HPC minicanda3
+    cd "${HOMEGFS}/ush/python/pygw"
+    python3 -m pip install .
+    #make links to the python packages used in the PR'ed repo
+    cd "${WF_ROOT_DIR}/ci/scripts"
+    if [[ ! -L workflow ]]; then
+      ln -s "${HOMEGFS}/workflow" workflow
+    fi
+    if [[ ! -L pygw ]]; then
+      ln -s "${HOMEGFS}/ush/python/pygw/src/pygw" pygw
+    fi
+    PSLOT=C96C48_hybatmDA # TODO loop over experments for each yaml file in the experiments dir
```

Comment:
Note on the TODO: We likely don't want to always run all of the defined experiments, so we'll want to have a list to test to run somewhere.

Reply:
I just coded this today to run through all the yaml files in the experiments dir instead, sense you mention that here, I'm going to update the PR with that update, I've all already tested it.